docs/api-docs/getting-started/api-status-codes.mdx

```diff
@@ -71,7 +71,7 @@ This article covers [REST](#rest-api-http-status-codes) and [GraphQL](#graphql-a
 |**400**|Invalid syntax, required data missing, `content-type` header missing|Double-check request body for syntax errors and missing data; check `content-type` header.|
 |**401** |API credentials are missing or invalid.|Double-check the `access_token` and `client_id`.|
 |||Send cURL request with the same credentials to rule app or config issues.|
-|**403**| App lacks required OAuth scopes, a store-owner account changed, operations resulting from API request exceed a platform limit, or URL requested is incorrect.|Double-check OAuth Scopes in **control panel** > **API Accounts** or in **Developer Portal** > **My Apps**.|
+|**403**| App lacks required OAuth scopes, a store-owner account changed, operations resulting from API request exceed a platform limit, or URL requested is incorrect. Account for a user authorized to install or uninstall apps changed. |Double-check OAuth Scopes in **control panel** > **API Accounts** or in **Developer Portal** > **My Apps**.|
```

Comment:
@piotrekkaminski I saw that previously, if a store owner account changed, users could get a 403 error. If an account changes for a user authorized to install/uninstall apps, can this also result in a 403 error?

Reply:
@Cdestewart do you know?

Reply:
@bc-andreadao can you elaborate on what you mean by a user account change? Do you mean if the user is added to a new account / removed from an old one? 

The install permissions are done on a store level so each store owner would need to grant the user permissions to be able to install / uninstall / launch an app.

Reply:
@Cdestewart Yes, I mean if the user is added to a new account / removed from an old one. I can reword my changes to make it more clear 

Would this throw a 403 error? (It wasn't explicitly on my ticket, so I wasn't sure. I originally added this change because the docs say that the 403 error can be thrown when a "store-owner account changed")

Reply:
They will get a 403 on any store they don't have the install/launch/uninstall permissions for so when they are added to a new account they will need to explicitly be granted the permissions.

Reply:
Got it, thanks, just updated the doc